toolkit/tools/srpmdownloader/srpmdownloader.go

```diff
@@ -0,0 +1,233 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
```

Comment:
Any tests would require setting some sort of localhost server as such: https://medium.com/zus-health/mocking-outbound-http-requests-in-go-youre-probably-doing-it-wrong-60373a38d2aa . But not sure how to guarantee security. Also may be better suited for network.go tests